apps/docs/src/routes/package/[name]/+page.server.ts

```diff
@@ -0,0 +1,16 @@
+import { error } from "@sveltejs/kit";
+import { mdToSvelte } from "@sveltepress/vite";
```

Comment:
This seems to be causing the error. It appears to be designed for Vite (to run during the build process) and is likely not intended to be included in the runtime build.

Reply:
Oh, sweet as... thanks for the lead!

Reply:
I took the opportunity to attempt to solve this case. The issue is that `win32` export doesn't exist in `pathe`. So, I created a PR to provide this export. This solved the issue with `pnpm run build` locally.

**To sum up:**

I know for sure that we need `pathe` to be used in the browser.

Do I understand correctly that for Cloudflare Pages we also need to use `pathe`⁣ — even inside the Vite build process?

Reply:
> Do I understand correctly that for Cloudflare Pages we also need to use pathe⁣ — even inside the Vite build process?

I tested—it worked! The answer is **no**.